core/dbt/contracts/graph/unparsed.py

```diff
@@ -661,6 +661,60 @@ def validate(cls, data):
             raise ValidationError("Group owner must have at least one of 'name' or 'email'.")
 
 
+#
+# semantic interfaces unparsed objects
+#
+
+
+@dataclass
+class Entity(dbtClassMixin, Replaceable):
```

Comment:
There's a bunch more Replaceable classes that we can skip including here.

Reply:
Done in latest commit.